webapp/platform/client/src/client4.ts

```diff
@@ -2851,7 +2851,7 @@ export default class Client4 {
 
     // Emoji Routes
 
-    createCustomEmoji = (emoji: CustomEmoji, imageData: File) => {
+    createCustomEmoji = (emoji: PartialExcept<CustomEmoji, 'name' | 'creator_id'>, imageData: File) => {
```

Comment:
Here I took the info from the description https://github.com/Dschoordsch/mattermost/blob/58eea1cd322e09a327bafd321d560d45010b5708/api/v4/source/emoji.yaml#L21-L24